### ISSUE TITLE
Add a feature where only certain roles can choose music for the bot to play

### ISSUE THREAD

**Issue by MrCamel999** [ComputerMaster1st/RailgunVB#5]
The title says it all. This is a feature that all my server members THINK is in place, as I use the bot as a 24/7 music bot in a certain VC. Nobody knows how to operate it, so we all good, but I would like this feature to be implemented (to all music, not just playlists nd such) that way I am sure that nobody can mess with my server’s music. Thanks!

**Comment by ComputerMaster1st** [comment]
Music Role Locking isn't in place atm, but has been suggested before. This shouldn't be hard to implement. When would you like this implemented?

**Comment by MrCamel999** [comment]
Anytime is fine, but as soon as possible is when I would prefer it. Not a huge deal bc the people of my server don’t seem to know a help command exists... lol

**Comment by ComputerMaster1st** [comment]
Right, after giving some thought and planning on what assets I need, I may be able to implement this. There is one thing that's bugging me though, and that's whether the implementation I have in mind would work or not. In theory it should. I'll let you know if it's successful.
